content/dstu2/condition.md

```diff
@@ -166,5 +192,5 @@ Note: abatementDateTime is not supported for diagnosis
 [category]: http://hl7.org/fhir/DSTU2/valueset-condition-category.html
 [clinical status]: http://hl7.org/fhir/DSTU2/valueset-condition-clinical.html
 [condition-category]: http://hl7.org/fhir/condition-category
-[argonaut extension-codes]: http://argonaut.hl7.org/extension-codes
+[argonaut]: http://argonaut.hl7.org
```

Comment:
yeah point taken. But I think we should still link to the correct system that it relates to. Open for discussion if we want to hyperlink it or just keep it as a plain text.